src/components/applications/kanban/Consents.tsx

```diff
@@ -100,6 +100,10 @@ const Consents: React.FC<ConsentsProps> = ({ handleClose, dialog }) => {
     <ConsentsContainer>
       <Header>
         <Typography className='title'>OAuth Consents</Typography>
+        <ButtonBase onClick={handleClose}>
+          {dialog && <CloseIcon />}
+          {/* {!dialog && <Tooltip title="Go to Applications"><KeyboardBackspaceIcon /></Tooltip>} */}
```

Comment:
Please remove this comment here since we won't be using it.

Reply:
I updated it.